packages/jsii-pacmak/lib/targets/kotlin/kdocbuilder.ts

```diff
@@ -0,0 +1,46 @@
+import { CodeMaker } from 'codemaker'
+
+export class KDocBuilder {
+  private readonly chunks: string[] = [];
+
+  public push(
+    chunk: string | undefined,
+    prefix: string | undefined = undefined
```

Comment:
Could make the last argument optional (this way `undefined` is the default value).